src/frontend/state/GlobalState.tsx

```diff
@@ -410,6 +410,15 @@ class GlobalState extends PureComponent<Props> {
   ): Promise<void> => {
     console.log('refreshing')
 
+    let updates = this.state.gameUpdates
```

Comment:
moved this call to checkGameUpdates to the top, so it updates the library info before using it in case this is the call to `legendary list`